doc/doxygen-root/README.md

```diff
@@ -0,0 +1,60 @@
+# CBMC documentation
```

Comment:
Should this readme file be lifted to the top-level of the documentation folder, or does it conflict with the compilation of the documentation?

Reply:
I could go either way.  My intention is that this documents the process of building the documentation, and so doesn't belong in the documentation itself.  I'm leaving it out for now.